### DISCUSSION TITLE
What happened to .png textures in `meshroom_batch` ?

### DISCUSSION THREAD

**Discussion by informatter** [alicevision/Meshroom#2146] [Q&A] [answered]
Hi Guys,

I am using the binaries for Meshroom-2023.1.0. When I use `meshroom_batch`, the texture files are in `.exr` format which can't be read by any 3d viewers. Is there a way to have the texture files as `.png`'s again?

I was checking https://github.dev/alicevision/Meshroom/blob/develop/bin/meshroom_batch but I don't see any flags for this.

Currently all the models I can view are loaded without any textures. 

Any help on this matter will be great.

Thank you

**Comment by cbentejac** [comment, marked as answer]
Hi @informatter,

The default value for the textures' file type is indeed `.exr`, but you can get them as `.png` by overwriting the corresponding parameter with the following argument in `meshroom_batch`: `--paramOverrides Texturing_1.colorMapping.colorMappingFileType="png"`.

**Reply by informatter** [reply]
@cbentejac thanks! 👍
